Backend/controllers/userc.js

```diff
@@ -26,13 +39,23 @@ const createUser = async (req, res) => {
 const getUserById = async (req, res) => {
   try {
     const id = req.params.userId;
+    if (id < 0 || isNaN(id)) {
```

Comment:
The `id` parameter must be a `string`, and not an `int`. Correct it accordingly, and rewrite the check

Reply:
Sure